npm/webpack-dev-server/src/makeDefaultWebpackConfig.ts

```diff
@@ -24,9 +24,17 @@ export function makeDefaultWebpackConfig (
 
   debug(`Using HtmlWebpackPlugin version ${version} from ${importPath}`)
 
+  // Webpack 5 uses `emitOnErrors: true`
+  // Webpack <= 4 uses `noEmitOnErrors: false`
+  const emitOnErrorsProperty = {
+    [config.sourceWebpackModulesResult.webpack.majorVersion === 5 ? 'emitOnErrors' : 'noEmitOnErrors']:
```

Comment:
Do we have coverage for both Webpack 4 & 5? Not immediately obvious

Reply:
Yeah we test multiple versions for most frameworks, some that use Webpack 4 and some that use 5